build.gradle

```diff
@@ -88,7 +88,7 @@ task fatJar(type: Jar) {
     manifest {
         attributes 'Main-Class': 'com.bloxbean.cardano.client.cli.CLIMain'
     }
-    baseName = project.name + '-all'
+    archivesBaseName = project.name + '-all'
```

Comment:
@edridudi Can we ignore these changes as part of this PR? Too many files. Hard to review :)  Let's keep the change to files specific to this PR.

Reply:
of course ..